dll/win32/shell32/wine/shellpath.c

```diff
@@ -1110,14 +1110,14 @@ static const CSIDL_DATA CSIDL_Data[] =
     { /* 0x03 - CSIDL_CONTROLS (.CPL files) */
         &FOLDERID_ControlPanelFolder,
         CSIDL_Type_SystemPath,
-        NULL,
+        L"ControlPanelFolder",
```

Comment:
This entire struct is updated on the wine side, including `.parsing` entries, possibly they have another fix for this issue?

Reply:
Wine 8.0 which I tried from the Debian repositories does not recognize any shell links like `shell:::{GUID}` or eg. `shell:Personal`. Updating this structure from wine does not make sense, since the code using the new fields also needs to be transferred, and this is not my goal.